client/my-sites/plans/controller.jsx

```diff
@@ -120,9 +122,12 @@ export default {
 
 	features( context ) {
 		const domain = context.params.domain;
-		const comparePath = domain ? `/plans/compare/${ domain }` : '/plans/compare';
+		const feature = get( context, 'params.feature' );
+		let comparePath = domain ? `/plans/compare/${ domain }` : '/plans/compare';
 
-		// check against feature enum
+		if ( isValidFeatureKey( feature ) ) {
+			comparePath += '?feature=' + feature;
```

Comment:
In the future, it might be worth using the `url` module here to help build this, especially if we begin passing more than one param.